### PR TITLE
feat(core): allow to pass custom selectors

### PR DESCRIPTION
**Pull request by Johann-S**
/CC @XhmikosR for a review

Fixes: #3

### CODE REVIEW

File: tests/main.spec.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ describe('bsCustomInputFile', function () {
     it('should add bsCustomFileInput property', function () {
       bsCustomFileInput.init()
 
-      expect(input.bsCustomFileInput).to.not.undefined
+      expect(input.bsCustomFileInput).not.undefined
```
Comment (XhmikosR) [inline tests/main.spec.js]: Shouldn't this include `be`?

Comment (Johann-S) [reply tests/main.spec.js]: Hmm I think it should works too 🤔 but I don't which one is better

Comment (XhmikosR) [reply tests/main.spec.js]: Your call :)

**Comment by XhmikosR** [comment]
Feel free to squash @Johann-S

**Comment by Johann-S** [comment]
Thank you for your changes @XhmikosR it's better thanks to you 👍

**Pull request merged**